File: unix/boot/spp/rpp/ratlibr/scopy.r
include	defs

# scopy - copy string at from (i) to to (j)

   subroutine scopy (from, i, to, j)
   character from (ARB), to (ARB)
   integer i, j

   integer k1, k2

   k2 = j
   for (k1 = i; from (k1) != EOS; k1 = k1 + 1) {
      to (k2) = from (k1)
      k2 = k2 + 1
      }
   to (k2) = EOS

   return
   end

File: unix/boot/spp/rpp/ratlibf/scopy.f
      SUBROUTINE SCOPY (FROM, I, TO, J)
      INTEGER FROM (100), TO (100)
      INTEGER I, J
      INTEGER K1, K2
      K2 = J
      K1 = I
23000 IF (.NOT.(FROM (K1) .NE. -2))GOTO 23002
      TO (K2) = FROM (K1)
      K2 = K2 + 1
23001 K1 = K1 + 1
      GOTO 23000
23002 CONTINUE
      TO (K2) = -2
      RETURN
      END
